atlasdb-cli/src/main/java/com/palantir/atlasdb/cli/output/OutputPrinter.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.cli.output;
+
+import org.slf4j.Logger;
+import org.slf4j.helpers.MessageFormatter;
+
+public class OutputPrinter {
+    private Logger logger;
+
+    public OutputPrinter(Logger logger) {
+        this.logger = logger;
+    }
+
+    public void info(String message, Object... args) {
+        logger.info(message, args);
+        System.out.println(MessageFormatter.arrayFormat(message, args).getMessage());
+    }
+
+    public void error(String message, Object... args) {
+        logger.error(message, args);
+        System.err.println(MessageFormatter.arrayFormat(message, args));
```

Comment:
I think this should end with `.getMessage()`, as above? SLF4J's FormattingTuple doesn't override toString() so if I'm not wrong this will only log the object identity string i.e. `org.slf4j.helpers.FormattingTuple@12345678`